etc/CONFIGURE_MACOS.bash

```diff
@@ -22,7 +22,7 @@ read
 
 # Note: openssl no longer required
 # Apple's provided flex is 2.6.4, which is the same that is provided by brew
-PKGS+="wget libtool autoconf automake libtool libxml2 libewf json-c"
+PKGS+="wget libtool autoconf automake libtool libxml2 libewf json-c re2 abseil pkg-config"
```

Comment:
If you're requiring `pkg-config`, you can add `ax_pkg_check_modules.m4` to your `m4` directory and then use `PKG_CHECK_MODULES()` in `configure.ac` to check on the existence/presence of pkg-config-managed packages and gather up their different CFLAGS and LIBS directives into namespaced variables, so you can incorporate those into `Makefile.am`.

See https://autotools.info/pkgconfig/pkg_check_modules.html

Reply:
Yes. I'm in fact using this now. See 
https://github.com/simsong/be20_api/blob/0bd6a2a8a72dffcc867cef478a3f84cf306f131f/be20_configure.m4#L31-L40

And you know what? I just spent 8 hours trying to debug this.